src/components/UI/LanguageSelector/index.tsx

```diff
@@ -39,6 +40,7 @@ export default function LanguageSelector({
     es: '🇪🇸',
     fr: '🇫🇷',
     hu: '🇭🇺',
+    ml: '🇮🇳',
```

Comment:
I'm not sure about putting a flag here. India has wayyyyyyyyyyyyyyy too many languages.

Reply:
ml:"മ🇮🇳";

what about putting first letter of language name in that languages script before the nation flag for denoting the language (only for nations containing more than one official languages)??

Reply:
Yeah no imo it'll be better to just use the Indian flag.